lib/grpnames.gd

```diff
@@ -73,24 +93,24 @@
 ##  Remark:
 ##   The normal subgroup lattice of G does not contain
 ##   all the information needed for the normal subgroup lattice
-##   of G2 for a decomposition G = G1 x G2. However let 
+##   of G2 for a decomposition G = G1 x G2. However let
 ##   G = G1 x .. x Gr be finest direct product decomposition
 ##   then all Gi are normal subgroups in G. Thus all Gi occur in
-##   the set NormalSubgroups(G) and we may split G recursively 
+##   the set NormalSubgroups(G) and we may split G recursively
 ##   without recomputing normal subgroup lattices for factors.
 ##
 ##  Method to enumerate factorizations given the divisors:
 ##   Consider a strictly increasing chain
 ##     1  <  a1 < a2 < .. < a_n  <  A
-##   of positive divisors of an integer A. 
+##   of positive divisors of an integer A.
 ##   The task is to enumerate all pairs 1 <= i <= j <= n
 ##   such that a_i*a_j = A. This is done by
 ##
-##   i := 1;  
+##   i := 1;
 ##   j := n;
-##   while i <= j do                   
-##     while j > i and a_i*a_j > A do 
-##       j := j-1; 
+##   while i <= j do
+##     while j > i and a_i*a_j > A do
+##       j := j-1;
 ##     end while;
```

Comment:
First thing you should do is teach your editor not to do that (while on the long run, I would like to get rid of all trailing spaces, we should do so explicitly, and not accidentally).

Then you can use `git rebase -i` to start editing the commits on this branch, retroactively. To get rid of the space changes, the command `git add -p` can be handy; or if you are on Windows or Mac OS X, the free [SourceTree](https://www.sourcetreeapp.com/) GUI tool can help with that.

No, this is a very vague description. I'll try to give a more full blown explanantion later.